includes/admin/class-wc-rest-payments-reader-controller.php

```diff
@@ -34,6 +54,16 @@ public function register_routes() {
 				'permission_callback' => [ $this, 'check_permission' ],
 			]
 		);
+		// TODO Definitive route definition when we implement the API in #3021.
+		register_rest_route(
+			$this->namespace,
+			'/' . $this->rest_base . '/receipt/(?P<payment_id>\w+)',
```

Comment:
Since reader and receipt share a one-to-many relationship, does it make sense to have the route as `/receipts`?

Reply:
Done!